### PR TITLE
Skia Geo Tiff Provider with lib tiff

### PR DESCRIPTION
**Pull request by inforithmics**

### CODE REVIEW

**Comment by inforithmics** [comment]
Changes 
I had to reference libtiff for reading tiff images. The rest is done with skia sharp.

File: Mapsui.Rendering.Skia/Mapsui.Rendering.Skia.csproj [hunk 1]
```diff
@@ -9,6 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="BitMiracle.LibTiff.NET" Version="2.4.649" />
```
Comment (pauldendulk) [inline Mapsui.Rendering.Skia/Mapsui.Rendering.Skia.csproj:12]: We can not add this to core. Perhaps we should just add more nugets for dependencies like this.

File: Mapsui.Rendering.Skia/Provider/GeoTiffProvider.cs [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+using System.Linq;
+using System.Runtime.InteropServices;
+using BitMiracle.LibTiff.Classic;
+using Mapsui.Layers;
+using Mapsui.Providers;
+using Mapsui.Styles;
+using SkiaSharp;
+using Color = Mapsui.Styles.Color;
+
+namespace Mapsui.Rendering.Skia.Provider
```
Comment (pauldendulk) [inline Mapsui.Rendering.Skia/Provider/GeoTiffProvider.cs:16]: Large parts of this code do not depend on skia. I would prefer to keep it outside of the Mapsui.Rendering.Skia project.

Comment (inforithmics) [reply Mapsui.Rendering.Skia/Provider/GeoTiffProvider.cs:16]: I will move this into another project, Maybe Mapsui.Extensions (For Mapsui Components with additional dependencies?)

**Comment by inforithmics** [comment]
Somehow libtiff can not read x and y resolution for the sample image so I have to investigate this.

File: Mapsui.Rendering.Skia/Provider/GeoTiffProvider.cs (hunk 2 above)
Comment (inforithmics) [reply Mapsui.Rendering.Skia/Provider/GeoTiffProvider.cs]: Is now in Mapsui.Extensions

**Comment by pauldendulk** [comment]
A merge conflict has arisen.

**Review by pauldendulk — APPROVED** [review]
👍

**Pull request merged**